api/app/signals/apps/email_integrations/templates/email/signal_default.txt

```diff
@@ -0,0 +1,20 @@
+Geachte heer, mevrouw,
```

Comment:
We should run this text by the business, as a misconfiguration will lead to it being sent to reporters. That said, the text looks reasonable.

Reply:
Ok, let's do that. This is why I kept it as close to the "optional" mail template we already have.